### PR TITLE
 React to dnx, hosting di changes

### PR DESCRIPTION
**Pull request by pakrym**
@davidfowl @rynowak

### CODE REVIEW

**Comment by pakrym** [comment]
After rhttps://github.com/aspnet/Hosting/pull/425

**Comment by dougbu** [comment]
CI builds are failing for different reasons Travis errorred out out with `System.InvalidOperationException : Unable to resolve service for type 'Microsoft.Extensions.Logging.ILogger`1[Microsoft.AspNet.Authorization.DefaultAuthorizationService]' while attempting to activate 'Microsoft.AspNet.Authorization.DefaultAuthorizationService'.` and AppVeyor attempt didn't compile.

File: test/Microsoft.AspNet.Mvc.FunctionalTests/MvcTestFixture.cs [hunk 1]
```diff
@@ -67,7 +67,6 @@ public MvcTestFixture(object startupInstance)
             using (new CultureReplacer())
             {
                 _server = TestServer.Create(
-                    CallContextServiceLocator.Locator.ServiceProvider,
```
Comment (dougbu) [inline test/Microsoft.AspNet.Mvc.FunctionalTests/MvcTestFixture.cs]: We use this provider two other places in this fixture. Is changing just this line enough to get the functional tests working again?

**Comment by pakrym** [comment]
This one should fix aspnetci build, ILogger issue should be fixed by https://github.com/aspnet/Mvc/commit/d584bcabcfadccb9805034f03aa904bf597e7c9b

**Pull request merged**